swagger/v1/defence_organisation.json

```diff
@@ -8,15 +8,12 @@
   "type": "object",
   "definitions": {
     "id": {
-      "description": "unique identifier of defence_organisation known internally as laa_contract_number",
+      "description": "The unique identifier of the defence organisation known internally as laa contract number",
       "type": "string",
       "example": "d1edc824-1668-4efa-870d-70d4a4cdccf4"
```

Comment:
I'm not convinced that this is a UUID. I don't know how to confirm the format though. And may not be important if we're not planning on providing this data starting in the near future anyway

Reply:
I've fixed that here:

https://github.com/ministryofjustice/laa-court-data-adaptor/pull/603/commits/5d27267bbe3a2e12b0967c399d64328bcc82ce2e